science/limesuite/Portfile

```diff
@@ -74,6 +74,11 @@ variant wxgui description {Enable wxWidgets LimeSuiteGUI} {
     configure.args-delete   -DENABLE_GUI=OFF
     configure.args-append   -DENABLE_GUI=ON \
```

Comment:
There is also `.args-replace`.

Reply:
I know from a long time but this is from the port creation :)

Reply:
I don't always remember `.args-replace` ... mostly do `delete` then `append` as noted here. Probably my doing LOL. Feel free to fix it or not; functionality is the same either way.

Reply:
FYI:
`variable-replace old-item new-item` is not present on `portfile(7)`

Reply:
I'm not sure what you are referring to, but the comment I see says `# Deprecate -replace with only one argument`. Why would you use a single argument?

Anyway, it's not really important, both work and we never actively mass-changed any of those parameters in any ports. It's just a random side note which you may or may not take into account. It works fine without changing anything.